nginx/config-files/hydroshare-local-nginx.conf.template

```diff
@@ -37,20 +25,6 @@ server {
     # Compute whether we need to rewrite an address. Rewrite all erronious 
     # external addresses but avoid rewriting internal references (from django 
     # management commands). 
-     
```

Comment:
Is this block just not needed?  I don't know nginx confiugration well enough to feel confident in these changes.  I guess if it works for you locally that should be evidence enough.

Reply:
I was in the same boat! I don't now nginx well, but errors seemed to indicate that this block was a problem and when I remove it, nginx runs :)

Reply:
Ill get a hold of the error logs when building/running without these modifications and reference them here

Reply:
okay I remember now
on local builds, we `sed` the "hostname" into the nginx config
https://github.com/hydroshare/hydroshare/blob/develop/local-dev-first-start-only.sh#L253

Because `$external` will be true and `$http_host != "FQDN_OR_IP"` will also be true on local dev, 
line 52 (`rewrite ^ http://FQDN_OR_IP$request_uri permanent;`) will execute, so we always get redirected to the hostname of our local machine, and the requests never make it to the hydroshare container.

Reply:
As far as I can tell, the `$external` check was intended to allow for "testing of the nginx reverse proxy" but at least for me, it was actually preventing this very thing.
@cbcunc do you have any insight? This is all local (dev) nginx conf